domain/src/test/java/org/oppia/android/domain/devoptions/ModifyLessonProgressControllerTest.kt

```diff
@@ -308,6 +315,65 @@ class ModifyLessonProgressControllerTest {
     }
   }
 
+  @Test
+  fun markFirstAndFractionsTopicsCompleted_TopicsAreCompleted() {
+    modifyLessonProgressController.markMultipleTopicsCompleted(
+      profileId,
+      listOf(TEST_TOPIC_ID_0, FRACTIONS_TOPIC_ID)
+    )
+    val allTopics = retrieveAllTopics()
+    val firstTopic = allTopics[0]
+    val fractionsTopic = allTopics[2]
+    assertThat(firstTopic.topicId).isEqualTo(TEST_TOPIC_ID_0)
+    assertThat(firstTopic.storyList[0].chapterList[0].chapterPlayState)
+      .isEqualTo(ChapterPlayState.COMPLETED)
+    assertThat(firstTopic.storyList[0].chapterList[1].chapterPlayState)
+      .isEqualTo(ChapterPlayState.COMPLETED)
+    assertThat(fractionsTopic.topicId).isEqualTo(FRACTIONS_TOPIC_ID)
+    assertThat(fractionsTopic.storyList[0].chapterList[0].chapterPlayState)
+      .isEqualTo(ChapterPlayState.COMPLETED)
+    assertThat(fractionsTopic.storyList[0].chapterList[1].chapterPlayState)
+      .isEqualTo(ChapterPlayState.COMPLETED)
+  }
+
+  @Test
+  fun markFirstAndRatios2StoriesCompleted_StoriesAreCompleted() {
```

Comment:
```suggestion
  fun markFirstAndRatios2StoriesCompleted_bothStoriesAreCompleted() {
```

Reply:
Changed to `markFirstAndRatios2StoriesCompleted_bothStoriesAreCompleted`